views/location.handlebars

```diff
@@ -0,0 +1,19 @@
+<script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+<script src="./javascript/location.js"></script>
+<div class="home">
+	<a href="/locations">
+		<h4 class="sub-heading" id="locations">Most Visited Cities</h4>
+		<!--The div element for the map -->
+		<div id="map">
```

Comment:
does this need to reference data from Country Codes model at all, or is that all done in JS file?

Reply:
At this point, it should be handled in the JS file; however, I have made a note on this question and will correct if that changes along the way.